ArduCopter/config.h

```diff
@@ -499,9 +499,6 @@
 //////////////////////////////////////////////////////////////////////////////
 // PosHold parameter defaults
 //
-#ifndef POSHOLD_ENABLED
```

Comment:
Not really sure why you're removing this.  We'd like to be able to remove POSHOLD at some stage in the future, so having a define to phase it out seems reasonable?

Reply:
The ability to remove the POSHOLD isn't really being used by anyone so I think it just adds ugly lines to the code.  I think when it comes to us eventually being in the situation where we can merge Loiter and PosHold, having this define here isn't really going to be helpful.